src/chapters/control-flow/studio.rst

```diff
@@ -0,0 +1,113 @@
+Studio: Control Flow to Count Characters
+=========================================
+
+In this studio, you will write a program to count the number of times each
+character occurs in a string and then print the results to the console.
+
+You could prompt the user for a string, but for the sake of simplicity, you can
+start by hard-coding the string and storing it in a variable. Here’s a test
+string, for your convenience:
+
+   TODO: Decide which of these to use.
```

Comment:
i suggest using the lorem ipsum, since the second option may be confusing to non-native english speakers like the pirate speak

Reply:
I was thinking the same thing about "Lorem ipsum..."  How about we just replace it with a non-gibberish, non-old-english phrase?  Maybe a paragraph from unit 1 or one from earlier in Java chapter 2?

Reply:
sure, although lorem ipsum is very commonly used so introducing them to this filler text is a good idea

Reply:
How about one of these:
1) "There is a theory which states that if ever anyone discovers exactly what the Universe is for and why it is here, it will instantly disappear and be replaced by something even more bizarre and inexplicable. There is another theory which states that this has already happened."
2) "He decided to spend the night there. He saw to it that all the sheep entered through the ruined gate, and then laid some planks across it to prevent the flock from wandering away during the night. There were no wolves in the region, but once an animal had strayed during the night, and the boy had had to spend the entire next day searching for it."
3) "Mr. and Mrs. Dursley, of number four, Privet Drive, were proud to say that they were perfectly normal, thank you very much. They were the last people you'd expect to be involved in anything strange or mysterious, because they just didn't hold with such nonsense."
4) "If the product of two terms is zero then common sense says at least one of the two terms has to be zero to start with. So if you move all the terms over to one side, you can put the quadratics into a form that can be factored allowing that side of the equation to equal zero. Once you’ve done that, it’s pretty straightforward from there."

Reply:
Which one is your favorite?